packages/fx-core/package-lock.json

```diff
@@ -879,79 +884,6 @@
       "resolved": "https://registry.npmjs.org/@jsdevtools/ono/-/ono-7.1.3.tgz",
       "integrity": "sha512-4JQNk+3mVzK3xh2rqd6RB4J46qUR19azEHBneZyTZM+c456qOrbbM/5xcR8huNCCcbVt7+UmizG6GuUvPvKUYg=="
     },
-    "@microsoft/teamsfx-api": {
```

Comment:
`teamsfx-api` is removed by `npm run setup`?

Reply:
I guess it's miss-added by #3641 (https://github.com/OfficeDev/TeamsFx/pull/3641/files)
Ideally for development `core` refs `api` via relative path, so `api` should not be added into package lock.

Reply:
The same to vsc package.json in #3644 (https://github.com/OfficeDev/TeamsFx/pull/3644/files)